### ISSUE TITLE
Issue of eigen descriptor code

### ISSUE THREAD

**Issue by JinyongJeong** [ethz-asl/segmatch#17]
In this code of eigen descriotor, order of eigen value is e1<e2<e3. 

So, I think linearity should be calculated with (e3 - e2) / e3). 

is it right?

**Comment by exodaniel** [comment]
@rdube 

If I am not mistaken when reading [the descriptor code](https://github.com/ethz-asl/segmatch/blob/master/segmatch/src/descriptors/eigenvalue_based.cpp#L70), and based on Weinmann 2014 ("Hence, it follows that λ1 ≥ λ2 ≥ λ3 ≥ 0 holds for each 3D point X."), it seems that the current code is indeed wrong. 

@JinyongJeong thank you for this catch!

**Comment by rdube** [comment]
@JinyongJeong good catch and thank you for pointing it out! Although this feature is indeed falsely named it seemed to be descriptive and highly selected by the random forest. We will fix this when pushing following demonstrations and features. Thank you!
